config.js.tmpl

```diff
@@ -28,6 +28,9 @@ exports.systemEmail = 'jrandom@example.com';
 // of the navigation for the admin section.
 exports.signupEnabled = false;
 
+// Configure Geocoding API API key
+exports.googleMapsApiKey = 'AIzaSyB9r-02V9oy8iluVuLweHx6IFi_VuUMXVg';
```

Comment:
Hey @kwhinnery, I see that you removed this in 3dd6ca0.  Are you okay with having this in the history or does this mean you need to change your key?